draconic/interpreter.py

```diff
@@ -886,7 +897,7 @@ def _eval_lambda(self, node):
     def _eval_call(self, node):
         func = self._eval(node.func)
         args = tuple(self._starred_unwrap(node.args))
-        kwargs = dict(self._eval(k) for k in node.keywords)
+        kwargs = dict(self._star_starred_unwrap(((n.arg, n.value) for n in node.keywords)))
```

Comment:
```suggestion
        kwargs = dict(self._star_starred_unwrap((k.arg, k.value) for k in node.keywords))
```
Removed unnecessary parenthesis. Also I think keeping it as `k` reads better.